contrib/completion/bash/docker

```diff
@@ -1759,7 +1759,7 @@ _docker_daemon() {
 			_filedir
 			return
 			;;
-		--exec-root|--graph|-g)
+		--exec-root|--data-root)
```

Comment:
Probably best to remove it initially, it's easier to add later than to deprecate / remove